modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaStorageTypeAwareCheck.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.util.StringUtil;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Tamyris Bernardo
+ */
+public class UpgradeJavaStorageTypeAwareCheck extends BaseUpgradeCheck {
+
+	@Override
+	protected String format(
+			String fileName, String absolutePath, String content)
+		throws Exception {
+
+		String newContent = content;
+
+		Matcher interfaceDepreciatedMatcher =
+			_interfaceDepreciatedPattern.matcher(content);
+
+		if (interfaceDepreciatedMatcher.find()) {
+			String interfaceDepreciated = interfaceDepreciatedMatcher.group();
```

Comment:
@TamyrisTorres, please rename to `deprecatedInterface`

Reply:
And fix the bug

Reply:
resolved!